automatic/minikube/legal/VERIFICATION.txt

```diff
@@ -0,0 +1,22 @@
+VERIFICATION
+
+Verification is intended to assist the Chocolatey moderators and community
+in verifying that this package's contents are trustworthy.
+
+The embedded software have been downloaded from the listed download
+location on <https://github.com/kubernetes/minikube/releases/tag/v0.24.1>
```

Comment:
the download location doesn't match the actual location used for the executable.
Either this location needs to be changed, or the file will need to be downloaded from github.

Reply:
Github binary doesn't end in ".exe" and messes Get-RemoteFiles; the alternative link which I use is mentioned under __Distribution__ (*Binaries are available through GitHub or on Google Cloud Storage. The direct GCS links are*)

Reply:
Then the VERIFICATION.txt file needs to reflect which files on the github repo is equal to the downloaded binaries.